components/ActivityIndicator/README.md

```diff
@@ -11,10 +11,12 @@ api_doc_root: true
 # Activity Indicators
 
 <div class="article__asset article__asset--screenshot">
-  <img src="docs/assets/activity_indicator.png" alt="Activity Indicator" width="375">
+  <img src="docs/assets/activityindicator.gif" alt="Activity Indicator" width="284">
 </div>
 
-Activity indicators are visual indications of an app loading content. The Activity Indicator is a circular indicator that either rotates clockwise or fills to completion clockwise when displaying progress.
+Material Design progress indicators display the length of a process or express an unspecified wait
```

Comment:
I think we're dropping "Material Design" and it should just be "Material progress indicators..."?